blivet/devices/md.py

```diff
@@ -640,6 +651,15 @@ def __init__(self, name, **kwargs):
         self._size = self.currentSize
 
     @property
+    def isDisk(self):
+        # pylint: disable=bad-super-call
+        return super(MDRaidArrayDevice, self).isDisk
+
+    def partitionable(self):
```

Comment:
Forgot @property here. But I also don't see why they are needed, it'll just call super w/o them defined in the subclass.